### PR TITLE
Redaction and translation updated.

### PR DESCRIPTION
**Pull request by Josev47**
Enhanced redaction and translation, made a better explanation to enhance the understanding about the description, the instructions, the tests and the challenge seed.

- [ x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x ] My pull request targets the `master` branch of freeCodeCamp.
- [ x ] None of my changes are plagiarized from another source without proper attribution.
- [ x ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/spanish/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.spanish.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ tests:
   }
 </style>
 
-<h2>CatPhotoApp</h2>
+<h2 class="red-text">CatPhotoApp</h2>
```
Comment (lynxlynxlynx) [inline curriculum/challenges/spanish/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.spanish.md]: this change is wrong, it's up to the user to solve the challenge, not us.

**Review by lynxlynxlynx — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@raisedadead It appears this PR is attempting to add more content to this challenge, instead of just a direct translation of the challenge.  

I believe we need to have new additions only come from the English version first and then we create translations (automatically using the bot) and also auto-create PRs for each language which would go through the normal PR review process.

If I am mistaken, please let me know.

**Comment by lynxlynxlynx** [comment]
There's no new content here, just the common mistake of adding the solution to the seed.

**Comment by RandellDawson** [comment]
@lynxlynxlynx Sorry about that.  It appeared there was additions to the description section, but it appears those were just translations.  I think I may have mixed this one up with another PR.

**Comment by lynxlynxlynx** [comment]
Thanks, I've fixed the remaining issues.

**Pull request merged**